pootle/apps/pootle_translationproject/models.py

```diff
@@ -135,9 +133,28 @@ def for_user(self, user):
         :return: A filtered queryset with `TranslationProject`s for `user`.
         """
         if user.is_superuser:
-            return self.all()
+            return self.live()
+
+        return self.live().filter(project__disabled=False)
```

Comment:
I think we can live without `enabled()` for the time being, and address custom queryset method chaining in a separate ticket.